pushing_testing_files/run_form_tests.yml

```diff
@@ -36,8 +36,8 @@ jobs:
       - name: Set languages
         run: echo "EXTRA_LANGUAGES=${{ github.event.inputs.extra_languages }}" >> $GITHUB_ENV
         if: ${{ github.event.inputs.extra_languages != '' }}
-      - name: List languages
-        run: echo "Extra languages to test are $EXTRA_LANGUAGES"
+      - run: echo "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY" >> $GITHUB_ENV
+      - run: echo "Repo address is $REPO_URL"
```

Comment:
Not sure why "languages" code is replaced with "repo_url" code. Both of those serve different purposes, so shouldn't they both be here?

Reply:
The diff is a bit misleading. Previously there was a step (?) to set the languages, then a `run` to list the languages out. I removed the second (since I noticed they get listed usefully in the log anyway) and added a `run` to set the `REPO_URL`.